src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/register/PullRequestStore.java

```diff
@@ -29,6 +30,7 @@
      * In the case that Jenkins misses a pull request deleted webhook, the pr no longer exists in bbs and so fetching
      * from bbs will not return it (hence the pr in our store is not updated to closed).
      * REST layer can call upon this method for a customer to update a pr in store to close in such case.
+     *
```

Comment:
nit: After line 23 there should also be a new line (i can't add a comment there)